packages/ses/README.md

```diff
@@ -503,6 +502,153 @@ system generated other diagnostic information hidden in side tables. The tamed
 console uses these side tables to output more informative diagnostics.
 [Logging Errors](./src/error/README.md) explains the design.
 
+## Security claims and caveats
+
+Provided that the `ses` implementation and its trusted compute base are
+correct, we claim that a host program (Figure 1) can evaluate a guest program
+(`program`) in a compartment after `lockdown` and that the program:
+
+- will initially only have access to one mutable object, `globalThis`,
+- specifically cannot modify any primordial object that it shares with the host
+  program
+- cannot perform any I/O, apart from I/O performed within the trusted compute
+  base, directly or indirectly,
+- and specifically cannot observe the passage of time at any resolution.
+
+However, such a program can:
+
+- execute for an indefinite amount of time,
+- allocate arbitrary amounts of memory,
+- detect the platform endianness,
+- in some JavaScript engines, observe the contents of the stack.
+  This may include sensitive information about the layout of files on the host
+  disk.
+  In cases where the stack is data-dependent, a guest can infer the data.
+  SES occludes the stack on V8 and SpiderMonkey, but can not on
+  JavaScriptCore.
+
+```js
+// Claims, Figure 1
+lockdown();
+const compartment = new Compartment();
+compartment.evaluate(program);
+```
+
+The trusted compute base includes:
+
+- the host hardware,
+- the host operating system,
+- the process memory manager,
+- a minimal implementation of JavaScript conforming to ECMAScript 262 as of
```

Comment:
```suggestion
- a implementation of JavaScript conforming to ECMAScript 262 as of
```

Your remaining text says more clearly what the restriction we the spec actually is. Given that, I'd remove "minimal" here as that suggests a tighter restriction than we mean.